src/payment/coupon/data/reducers.js

```diff
@@ -0,0 +1,68 @@
+import { ADD_COUPON, REMOVE_COUPON, UPDATE_COUPON_DRAFT } from './actions';
+
+const defaultState = {
+  benefit: null,
```

Comment:
`benefit` describes what the coupon does to the price.  `code` is something like "SUMMER20".  `voucherId` is a numeric ID that describes the particular, applied coupon instance.

Reply:
Would these be useful comments to add to the code?